metrics/pipe_analysis.yaml

```diff
@@ -68,3 +68,26 @@ fake_stars_magDiffSigMad_base_CircAper_12_0_mag:
       - photometry
       - fakes
 
+skyObjectMean_CircApRad9pix:
+  description: >
+    The mean of all the sky object base_CircularApertureFlux_9_0_instFlux values.
+  unit: nanojansky
+  tags:
+  - skyObject
+  - skyLevel
+  - skyMean
```

Comment:
Are all of skyObject, skyLevel and skyMean really useful here (and below, skyMean in the Std definition)?

And, again, I think it may be a better idea to leave the specific flux algorithm from the name and you could indicate in the description that the particular flux measurement algorithm is included in the metadata.

Reply:
I've reduced the tags to just `- skyObject` and `- skyLevel` for both the sky object mean and standard deviation metric measures. 

(PS - Following conversations we've had over the prior couple of weeks and the issues surrounding shortening these metric names, I've left the metric names as-is for now)

Reply:
PPS: As a small update for the sake of accuracy, I've updated the two description strings to mention the fact that these are statistical values based on sigma-clipped data, i.e.:
```
The sigma-clipped mean for all sky object base_CircularApertureFlux_9_0_instFlux values.
The sigma-clipped standard deviation for all sky object base_CircularApertureFlux_9_0_instFlux values.
```